packages/kbn-rule-data-utils/src/technical_field_names.ts

```diff
@@ -28,6 +28,7 @@ const ALERT_DURATION = `${ALERT_NAMESPACE}.duration.us` as const;
 const ALERT_SEVERITY_LEVEL = `${ALERT_NAMESPACE}.severity.level` as const;
 const ALERT_SEVERITY_VALUE = `${ALERT_NAMESPACE}.severity.value` as const;
 const ALERT_STATUS = `${ALERT_NAMESPACE}.status` as const;
+const SPACE_IDS = `${ALERT_NAMESPACE}.space_ids` as const;
```

Comment:
As decided on Alerts as Data schema meeting, space IDs are now top-level fields: `kibana.space_ids`.

Reply:
It looks like it's actually been changed to `kibana.alert.space_ids - Field used by the RBAC code to filter by space_id.` here: https://docs.google.com/document/d/16Z5KTKZnGDIuwpvjJylKSYb1MJ9ThEqoF1pRhRWnkqo/edit#

I'll push up a change thanks @xcrzx!!

Reply:
We found a more updated doc with the new top-level field, will be using `kibana.space_ids` 👍